akka-persistence/src/main/scala/akka/persistence/PersistencePlugin.scala

```diff
@@ -86,6 +86,12 @@ private[akka] abstract class PersistencePlugin[ScalaDsl, JavaDsl, T: ClassTag](s
           instantiate((classOf[ExtendedActorSystem], system) :: (classOf[Config], pluginConfig) :: Nil)
       }
       .recoverWith { case _: NoSuchMethodException => instantiate((classOf[ExtendedActorSystem], system) :: Nil) }
+      .recoverWith {
+        case _: NoSuchMethodException =>
+          instantiate(
+            (classOf[ExtendedActorSystem], system) :: (classOf[Config], pluginConfig) ::
+            (classOf[String], configPath) :: (classOf[Config], mergedConfig) :: Nil)
+      }
       .recoverWith { case _: NoSuchMethodException => instantiate(Nil) }
```

Comment:
we would support constructors:
1. (system, pluginConfig, configPath)
1. (system, pluginConfig)
1. (system)
1. (system, pluginConfig, configPath, mergedConfig)

We could document what these mean but it's not an obviously clear api.